Added fix for AC_REPLACE_FNMATCH in configure.ac

    On a x86-64 target, AC_REPLACE_FNMATCH led to the following error
    duing the configure step:

    | config.status: linking ././fnmatch_.h to ./fnmatch.h
    | config.status: error: ././fnmatch_.h: file not found
    | ERROR: oe_runconf failed

    AC_REPLACE_FNMATCH is now deprecated as of AC 2.61. It is replaced
    with AC_FUNC_FNMATCH for the time being.
 
Upstream Status: Pending

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index d5f7812..c05691c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,7 +88,7 @@ AC_C_VOLATILE
 # Checks for library functions.
 AC_FUNC_CLOSEDIR_VOID
 AC_FUNC_ERROR_AT_LINE
-AC_REPLACE_FNMATCH
+AC_FUNC_FNMATCH
 AC_FUNC_FORK
 AC_PROG_GCC_TRADITIONAL
 AC_FUNC_MALLOC

